build.gradle

```diff
@@ -66,6 +66,7 @@ dependencies {
 	implementation('net.logstash.logback:logstash-logback-encoder:5.2')
 
 	implementation('org.camunda.bpm:camunda-engine-spring:7.9.0')
+	implementation('com.fasterxml.uuid:java-uuid-generator:3.1.0')
```

Comment:
There's a 4.0.1, a 3.2.0 and a 3.3.0 in Maven, are we deliberately using an older version?

Reply:
Yep, 3.1 was listed as this is what camunda 7.9 is using. I've updated the version to 3.1.2 to match exactly what Camunda 7.9 spec says.